doc/crds/ip-reconciler-job.yaml

```diff
@@ -24,6 +24,7 @@ spec:
               command:
                 - /ip-reconciler
                 - -kubeconfig=/host/etc/cni/net.d/whereabouts.d/whereabouts.kubeconfig
+                - -log-level=debug
```

Comment:
I've added this to preserve the current behavior, but I'm not sure we want it to be this verbose. 

Your thoughts ?